frontend/src/components/messages/RequestConversations.vue

```diff
@@ -0,0 +1,248 @@
+      type: [String, null],
+      required: true,
+      default: ''
+    }
+  },
+  emits: ['toggleMarkUnreadButtonInConversationThread'],
+  data() {
+    return {
+      format,
+      showReplyRequestDialog: false,
+      isSortedDesc: true,
+      assistanceRequest: null,
+      headers: [
+        {
+          title: 'dateReceived',
+          key: 'dateReceived',
+          sortable: true
+        }
+      ],
+    }
+  },
+  computed: {
+    ...mapState(useMessagesStore, ['assistanceRequests', 'assistanceRequestConversation']),
+    ...mapState(useAuthStore, ['userInfo']),
+    isReplyDisabled() {
+      return this.assistanceRequest &&
+        (this.assistanceRequest.statusCode !== ASSISTANCE_REQUEST_STATUS_CODES.WITH_PROVIDER &&
+          this.assistanceRequest.statusCode !== ASSISTANCE_REQUEST_STATUS_CODES.READY_TO_RESOLVE);
+    }
+  },
+  watch: {
+    // When assistanceRequestId changes, get the conversation for the new assistance request.
+    assistanceRequestId: async function (newVal) {
+      await this.getAssistanceRequestConversation(this.assistanceRequestId)
+      this.assistanceRequest = this.assistanceRequests.find(item => item.assistanceRequestId === newVal)
+    },
+  },
+  methods: {
+    ...mapActions(useMessagesStore, ['getAssistanceRequestConversation']),
+
+    /**
+     * Sorts the conversation by date received based on isSortedDesc.
+     */
+    sortConversation() {
+      this.assistanceRequestConversation.sort((a, b) => {
+        if (this.isSortedDesc) {
+          return (a.dateReceived < b.dateReceived) ? 1 : -1;
+        } else {
+          return (a.dateReceived > b.dateReceived) ? 1 : -1;
+        }
+      });
+    },
+
+    /**
+     * Toggles the sort order of the conversation.
+     */
+    toggleSort() {
+      this.isSortedDesc = !this.isSortedDesc
+      this.sortConversation()
+    },
+
+    /**
+     * Toggles the reply request dialog opened/closed.
+     */
+    toggleReplyRequestDialog() {
+      this.showReplyRequestDialog = !this.showReplyRequestDialog
+    },
+
+    /**
+     * Called when the reply request dialog emits a reply-success-event. Gets the conversation for the
+     * assistance request and sorts them.
+     */
+    async replySuccessEvent(isSuccess) {
+      if (isSuccess) {
+        await this.getAssistanceRequestConversation(this.assistanceRequestId);
+        this.sortConversation();
+        this.setSuccessAlert('Reply sent successfully');
+      }
+    },
+
+    /**
+     * Returns the text to display in the tooltip for the reply button when it is disabled.
+     */
+    getReplyDisabledText() {
+      return ASSISTANCE_REQUEST_REPLY_DISABLED_TEXT;
+    },
+
+    /**
+     * Returns the sent by value for the conversation depending on the source system.
+     */
+    deriveFromDisplay(item) {
+      return (item.ofmSourceSystem) ? this.userInfo?.firstName + ' ' + this.userInfo?.lastName : OFM_PROGRAM
+    }
+
+  }
+}
+</script>
+  
+<style scoped>
+:global(.tooltip) {
```

Comment:
If this is a shared style it would be better in main.css rather than using the global selector.

Reply:
Fixed.